Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -426,8 +434,30 @@ void FieldProbe::ComputeDiags (int step)
         for (MyParIter pti(m_probe, lev); pti.isValid(); ++pti)
         {
             const auto getPosition = GetParticlePosition(pti);
-            auto const np = pti.numParticles();
+                auto setPosition = SetParticlePosition(pti);
```

Comment:
keep this alignment left on the line:
```suggestion
            auto setPosition = SetParticlePosition(pti);
```